caluma/tests/__snapshots__/test_schema.ambr

```diff
@@ -1,11 +1,6 @@
-# name: test_schema_introspect
```

Comment:
@fugal-dy give this a closer look!

Reply:
@winged not sure why this shows diff to main branch rather than upstream/graphene_django_v3

a local comparison of the files reveals that only changes re: workitem`s `redo`-pattern related elements are added as well as `addressedGroups`, `responsibleUsers` and `controllingGroups` get comments and questions `hint_text`s

The test_schema_introspect seems to maintain the correct result and the workaround introduced before is no longer needed.

correct me if I'm wrong.